web/packages/teleport/src/Roles/RoleEditor/EditorHeader.tsx

```diff
@@ -17,9 +17,9 @@
  */
 
 import React from 'react';
-import { Flex, ButtonText, H2, Indicator, Box } from 'design';
+import { Flex, H2, Indicator, Box, ButtonIcon } from 'design';
 import { HoverTooltip } from 'design/Tooltip';
-import { Trash } from 'design/Icon';
+import * as Icon from 'design/Icon';
```

Comment:
```suggestion
import { Cross, Trash } from 'design/Icon';
```

we don't need to import all right?

Reply:
We don't, but using `Icon` as an alias for that module is simply a convention that we seem to use everywhere else.